<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4" alignment="Qt::AlignLeft">
     <widget class="QTableView" name="tableView_2"/>
    </item>
    <item row="5" column="0" rowspan="2" alignment="Qt::AlignLeft">
     <widget class="QListView" name="attachments_listview"/>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="to_textbox"/>
    </item>
    <item row="2" column="1" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Subject:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>To :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="subject_textbox"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="delete_attachment">
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="file_explorer_btn">
      <property name="text">
       <string>Add file</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="4" colspan="3">
     <widget class="QTextEdit" name="email_conten_box"/>
    </item>
   </layout>
   <zorder>label</zorder>
   <zorder>to_textbox</zorder>
   <zorder>label_2</zorder>
   <zorder>subject_textbox</zorder>
   <zorder>email_conten_box</zorder>
   <zorder>tableView</zorder>
   <zorder>tableView_2</zorder>
   <zorder>attachments_listview</zorder>
   <zorder>delete_attachment</zorder>
   <zorder>file_explorer_btn</zorder>
   <zorder>sendButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
